testing/shared/src/main/scala/io/iteratee/testing/IterateeSuite.scala

```diff
@@ -61,7 +61,7 @@ abstract class IterateeErrorSuite[F[_], T: Arbitrary: Eq: Cogen](implicit
   "ensureEval" should "be executed when the iteratee is done" in forAll { (eav: EnumeratorAndValues[Int]) =>
     var done = false
 
-    val iteratee = consume[Int].ensureEval(Eval.always(F.pure(done = true)))
+    val iteratee = consume[Int].ensureEval(Eval.always(F.pure { done = true }))
```

Comment:
This is to fix a deprecation warning about assignments looking like named arguments.